README.md

```diff
@@ -1,7 +1,13 @@
 # PuppetDB
 
-[docs]: https://docs.puppetlabs.com/puppetdb/latest
+[docs]: https://docs.puppet.com/puppetdb/latest
 
 PuppetDB is the fast, scalable, and reliable data warehouse for Puppet. It caches data generated by Puppet, and gives you advanced features at awesome speed with a powerful API.
 
 For documentation on this product, consult the [latest documentation][docs].
+
+## Maintenance
+
+Maintainers: Ryan Senior <ryan.senior@puppet.com>, Wyatt Alt <wyatt@puppet.com>, Rob Browning <rlb@puppet.com>, AJ Roetker <andrew.roetker@puppet.com>, Russell Mull <russell.mull@puppet.com>, Ken Barber <ken@puppet.com>
```

Comment:
Agreed - this is a bad fit for projects with well-established practices. How about you add a sentence to this section along the lines "For questions/problems/dev help email <our public mailing list>" and then as maintainers list one or two of the people working on PuppetDB. The reason for listing people is to give others a way to get in touch with a human if they can't get attention through other means. In that sense, the maintainer duties are mostly around redirecting and connecting people with the right devs for whatever their problems are.